index.html

```diff
@@ -17,47 +17,45 @@ <h1 class="title">INTENTION TIMER</h1>
         <h2 class="subhead">New Activity</h2>
           <section class="category-timer-wrapper">
             <article class="timer-box-wrapper hidden">
-            <!-- <article class="timer-box-wrapper"> -->
               <h3 class="timer-box-header">Deep Breathing</h3>
-              <h1 class="countdown-clock">05:00</h2>
+              <h1 class="countdown-clock">05:00</h1>
               <div class="start-circle-wrapper">
                 <p class="start-circle-text">START</p>
               </div>
             </article>
-            <!-- <form class="form hidden"> -->
             <form class="form">
               <article class="categories">
                 <h3 class="form-label">Select a category:</h3>
                 <section class="category-box-wrapper">
                   <div class="category-boxes study-box">
                     <img class="category-image study-image" src="./assets/study.svg" alt="light bulb above an open book">
```

Comment:
Might consider changing consistent with "Option 1 - select study activity", "Option 2 - select meditate activity", et al.

Reply:
Right on. Done and pushed.